src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -103,14 +104,14 @@ const myAppConfig = {
 
 Amplify.configure(myAppConfig);
 ```
-
+[comment]: # (RENE, I realize this is likely outside the scope of this review, but the message in line 101 above within the code block (Before calling API,graphql...) isn't particularly clear. Is this something we can update to make clearer for the user?)
 </Block>
 </BlockSwitcher>
 
 #### Using with an AppSync custom domain name
 
-[Custom domain names](https://docs.aws.amazon.com/appsync/latest/devguide/custom-domain-name.html) can be used to create a memorable endpoint with a domain name of your choice. These can have any format, but must end with `/graphql` (see https://graphql.org/learn/serving-over-http/#uris-routes). Using custom domain names will impact how you enter your graphqlEndpoint:
-
+[Custom domain names](https://docs.aws.amazon.com/appsync/latest/devguide/custom-domain-name.html) can be used to create a memorable endpoint with a domain name of your choice. These can have any format, but must end with `/graphql` (see [URIs, Routes](https://graphql.org/learn/serving-over-http/#uris-routes) on the GraphQL website). Using custom domain names will impact how you enter your graphqlEndpoint:
+[question]: # (RENE, In the previous line, can we be more specific about how use of the custom domain name affects how you enter your graphqlEndpoint? How is this demonstrated in the following code?)
```

Comment:
Q from Jim - In the previous line, can we be more specific about how use of the custom domain name affects how you enter your graphqlEndpoint? How is this demonstrated in the following code?